lib/legally.js

```diff
@@ -5,7 +5,12 @@ var open = require('./open');
 var folders = require('./folders');
 
 function getPackage(path){
-  var name = path.split(/\//).pop();
+  var name;
+  if (path.indexOf("@types") > 0) {
```

Comment:
Is this intended to be `if (path.indexOf("@types") > -1) {`? (0 is a possible index, it gives -1 when it was not found)